tests/load/pipeline/test_clickhouse.py

```diff
@@ -78,3 +97,135 @@ def items2() -> Iterator[TDataItem]:
     finally:
         with pipeline.sql_client() as client:
             client.drop_dataset()
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True, subset=["clickhouse"]),
+    ids=lambda x: x.name,
+)
+def test_clickhouse_destination_merge_dicts(
```

Comment:
please move that test to `test_merge_disposition` and configure it like any other test there. also run with both merge strategies. we need to makes sure we do not have similar problems elsewhere